packages/create-next-app/helpers/examples.ts

```diff
@@ -78,7 +78,7 @@ export function downloadAndExtractRepo(
     ),
     tar.extract(
       { cwd: root, strip: filePath ? filePath.split('/').length + 1 : 1 },
-      [`${name}-${branch}${filePath ? `/${filePath}` : ''}`]
+      [`${name}-${branch.replace(/\//, '-')}${filePath ? `/${filePath}` : ''}`]
```

Comment:
```suggestion
      [`${name}-${branch.replace(/\//g, '-')}${filePath ? `/${filePath}` : ''}`]
```
Looks like we might need to handle multiple slashes